pkg/operator/ceph/cluster/rbd/config.go

```diff
@@ -59,6 +59,16 @@ func (r *ReconcileCephRBDMirror) generateKeyring(clusterInfo *client.ClusterInfo
 		return "", err
 	}
 
+	if r.shouldRotateCephxKeys {
+		logger.Infof("rotating cephx key for CephRBDMirror %q in the namespace %q", daemonConfig.ResourceName, clusterInfo.Namespace)
+		newKey, err := s.RotateKey(user)
+		if err != nil {
+			return "", errors.Wrapf(err, "failed to rotate cephx key for CephRDBMirror %q in the namespace %q", daemonConfig.ResourceName, clusterInfo.Namespace)
+		} else {
```

Comment:
Remove else as we are returning inside if

Reply:
done.